apps/wallet/frontend/src/components/ListAllocationRequests.tsx

```diff
@@ -135,16 +142,34 @@ const AllocationRequestActionButton: React.FC<{
       console.error('Failed to submit allocation', error);
     },
   });
+  const withdrawAllocationMutation = useMutation({
+    mutationFn: async () => {
+      if (correspondingAllocation) {
+        return await withdrawAllocation(correspondingAllocation.contractId);
+      } else {
+        throw new Error("This mutation shouldn't be called without a corresponding allocation");
+      }
+    },
+    onSuccess: () => {},
+    onError: error => {
+      console.error('Failed to withdraw allocation', error);
+    },
+  });
 
   if (!actionAllowed) return null;
-  // TODO (#1413): show the withdraw button and implement the callback, instead of showing nothing
-  if (alreadyAccepted) return null;
-  // return (
-  //   <Button variant="pill" size="small" className="allocation-request-withdraw">
-  //     Withdraw
-  //   </Button>
-  // );
-  else
+  if (alreadyAccepted) {
+    return (
+      <Button
```

Comment:
iirc we usually disable the button while the mutation is running? any reason not to do that here?